bench/property-getter-access.js

```diff
@@ -0,0 +1,185 @@
+const Benchmark = require('benchmark');
+const suite = new Benchmark.Suite();
+const { eventToMdTable, H2, createTableHeader } = require('../markdown');
+
+const tableHeader = createTableHeader(['name', 'ops/sec', 'samples']);
+
+class TestGetter {
+  get test1() {
+    return 'Hello';
+  }
+}
+
+const classGetter = new TestGetter();
+
+const getterObj = {
+  get test2() {
+    return 'Hello';
+  },
+};
+
+const methodObj = {
+  test3() {
+    return 'Hello';
+  },
+};
+
+const defineObj = {};
+
+Object.defineProperty(defineObj, 'test4', {
+  get() {
+    return 'Hello';
+  },
+});
+
+const defineEnumerableFalseObj = {};
+
+Object.defineProperty(defineEnumerableFalseObj, 'test5', {
+  get() {
+    return 'Hello';
+  },
+  enumerable: false,
+});
+
+const defineConfigFalseObj = {};
+
+Object.defineProperty(defineConfigFalseObj, 'test6', {
+  get() {
+    return 'Hello';
+  },
+  enumerable: true,
+  configurable: false,
+});
+
+const defineEnumerableFalseAndConfigFalseObj = {};
+
+Object.defineProperty(defineEnumerableFalseAndConfigFalseObj, 'test7', {
+  get() {
+    return 'Hello';
+  },
+  enumerable: false,
+  configurable: false,
+});
+
+const defineOnlyWritableObj = {};
+
+Object.defineProperty(defineOnlyWritableObj, 'test8', {
+  writable: true,
+  value: 'Hello',
+});
+
+const defineWritableEnumerableFalseObj = {};
+
+Object.defineProperty(defineWritableEnumerableFalseObj, 'test9', {
+  writable: true,
+  value: 'Hello',
+  enumerable: false,
+});
+
+const defineWritableEnumerableFalseAndConfigFalseObj = {};
+
+Object.defineProperty(defineWritableEnumerableFalseAndConfigFalseObj, 'test10', {
+  writable: true,
+  value: 'Hello',
+  enumerable: false,
+  configurable: false,
+});
+
+const defineWithTwoPropertiesObj = {};
+
+Object.defineProperty(defineWithTwoPropertiesObj, 'test11', {
+  get() {
+    return 'Hello';
+  },
+});
+
+Object.defineProperty(defineWithTwoPropertiesObj, 'world', {
+  get() {
+    return 'Hello';
+  },
+});
```

Comment:
I add twice to be fair with the comparison, I compare this getter access with another object that has two properties too.

Reply:
Actually, I just add to make sure that getting value from an object with two properties doesn't have any difference, but I removed it because it doesn't make too much sense.